quantities/markup.py

```diff
@@ -88,7 +88,7 @@ def format_units_latex(udict,font='mathrm',mult=r'\\cdot',paren=False):
     '''
     Replace the units string provided with an equivalent latex string.
 
-    Division (a/b) will be replaced by \frac{a}{b}.
+    Division (a/b) will be replaced by \\frac{a}{b}.
```

Comment:
This has not been fixed yet

https://github.com/python-quantities/python-quantities/blob/269d81e8f1fa308b5a0701b8136fd642153e9534/quantities/markup.py#L91